commands/new_function.go

```diff
@@ -46,10 +47,38 @@ language or type in --list for a list of languages available.`,
 	RunE:    runNewFunction,
 }
 
+func containsLowerCase(funcname string) bool {
```

Comment:
We could perhaps use a regex here?

What are the valid names of services on Kubernetes? They are in this regex:

https://github.com/openfaas/faas-netes/blob/04888f560506c452ed684c4d376ff08fbfe5daea/handlers/deploy.go#L39

Is this regex also compatible with the "image" field for Docker image names?